api/src/main/resources/liquibase-schema-only.xml

```diff
@@ -2648,7 +2654,7 @@
         <addForeignKeyConstraint baseColumnNames="cohort_id" baseTableName="cohort_member" constraintName="parent_cohort" deferrable="false" initiallyDeferred="false" referencedColumnNames="cohort_id" referencedTableName="cohort"/>
     </changeSet>
     <changeSet author="ben (generated)" id="1227303685425-112">
-        <addForeignKeyConstraint baseColumnNames="patient_id" baseTableName="cohort_member" constraintName="member_patient" deferrable="false" initiallyDeferred="false" onUpdate="CASCADE" referencedColumnNames="patient_id" referencedTableName="patient"/>
```

Comment:
You can't edit an existing changeset, you need to add a new changeset to the liquibase-update-to-latest.xml file that drops this